doc/man/scons.xml

```diff
@@ -6703,23 +6706,66 @@ to enclose a Python expression to be evaluated.
 See <xref linkend='python_code_substitution'/> below
 for a description.</para>
 
-<para>A variable name
-may also be a Python function
-associated with a
-&consvar; in the environment.
-The function should
-accept four arguments:
+<para>The special pseudo-variables
+<emphasis role="bold">$(</emphasis>
+and
+<emphasis role="bold">$)</emphasis>
+may be used to surround parts of a command line
+that may change
+<emphasis>without</emphasis>
+causing a rebuild--that is,
+which are not included in the signature
+of target files built with this command.
+All text between
+<emphasis role="bold">$(</emphasis>
+and
+<emphasis role="bold">$)</emphasis>
+will be removed from the command line
+before it is added to file signatures,
+and the
+<emphasis role="bold">$(</emphasis>
+and
+<emphasis role="bold">$)</emphasis>
+will be removed before the command is executed.
+For example, the command line:</para>
+
+<programlisting language="python">
+echo Last build occurred $( $TODAY $). &gt; $TARGET
+</programlisting>
+
+<para>would execute the command:</para>
+
+<screen>
+echo Last build occurred $TODAY. &gt; $TARGET
+</screen>
+
+<para>but the command signature added to any target files would be:</para>
+
+<screen>
+echo Last build occurred  . &gt; $TARGET
+</screen>
+
+<para>A variable name may also be the name
```

Comment:
Kinda awkward wording here..
A referenced variable can refer to a python function... ?

Reply:
it *is* awakward.  I was trying to arrange it better than it was but it's still hokey.  $foo doesn't refer directly to a function, it is the name of a construction variable whos value is the name of a function.  not sure there's any way to say that simply...

Reply:
`env['foo']=my_func` and then `$foo` would work.

Reply:
reworded